### PR TITLE
Fixed auto clear up of filled information if task creation fails

### PR DESCRIPTION
**Pull request by DeRaowl**
- Fixed auto-clear up of filled information if task creation fails
https://user-images.githubusercontent.com/26904087/152802423-e9c9db05-a750-45b5-b6e4-947ea43f4d98.mp4

### CODE REVIEW

File: src/task/script.js [hunk 1]
```diff
@@ -206,8 +206,10 @@ taskForm.onsubmit = async (e) => {
 
     const result = await response.json();
 
+    if (response.ok) {
+      window.location.reload(true);
+    }
     alert(result.message);
```
Comment (Pavangbhat) [inline src/task/script.js:212]: What if the task was created successfully?
Will the user get a response message shown?

**Review by Pavangbhat — CHANGES_REQUESTED** [review]

File: src/task/script.js (hunk 1 above)
Comment (DeRaowl) [reply src/task/script.js:212]: Yes, Only reload has been avoided. Users receive messages after successful task creation.

Comment (Pavangbhat) [reply src/task/script.js:212]: No, I don't think so 
Here line 210 will be executed if the response is successful which reloads the page so line 212 will not be shown to the user.
Please fix it

**Review by Pavangbhat — APPROVED** [review]

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Some of the checks are failing 
Please check why they are failing and push the changes

**Pull request merged**